### ISSUE TITLE
HTTP 400: Bad Request retrieving all uploaded albums

### ISSUE THREAD

**Issue by Sunii** [apastel/ytmusic-deleter#9]
Running "ytmusic-deleter delete-uploads" ends in HTTP 400: Bad Request

Windows 10
ytmusic-deleter, version 1.2.1

`PS C:\WINDOWS\system32> ytmusic-deleter delete-uploads
[2021-01-23 11:56:40] Retrieving all uploaded albums...
Traceback (most recent call last):
  File "c:\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Python39\Scripts\ytmusic-deleter.exe\__main__.py", line 7, in <module>
  File "c:\python39\lib\site-packages\click\core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "c:\python39\lib\site-packages\click\core.py", line 782, in main
    rv = self.invoke(ctx)
  File "c:\python39\lib\site-packages\click\core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "c:\python39\lib\site-packages\click\core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\python39\lib\site-packages\click\core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "c:\python39\lib\site-packages\ytmusic_deleter\cli.py", line 46, in delete_uploads
    (albums_deleted, albums_total) = delete_uploaded_albums(add_to_library)
  File "c:\python39\lib\site-packages\ytmusic_deleter\cli.py", line 60, in delete_uploaded_albums
    uploaded_albums = youtube_auth.get_library_upload_albums(sys.maxsize)
  File "c:\python39\lib\site-packages\ytmusicapi\mixins\uploads.py", line 88, in get_library_upload_albums
    get_continuations(results, 'gridContinuation', limit - len(albums), request_func,
  File "c:\python39\lib\site-packages\ytmusicapi\parsers\utils.py", line 107, in get_continuations
    response = request_func(additionalParams)
  File "c:\python39\lib\site-packages\ytmusicapi\mixins\uploads.py", line 84, in <lambda>
    request_func = lambda additionalParams: self._send_request(
  File "c:\python39\lib\site-packages\ytmusicapi\ytmusic.py", line 116, in _send_request
    raise Exception(message + error)
Exception: Server returned HTTP 400: Bad Request.
Request contains an invalid argument.
PS C:\WINDOWS\system32> ytmusic-deleter --version
ytmusic-deleter, version 1.2.1
PS C:\WINDOWS\system32>`

**Comment by apastel** [comment]
Usually when these types of errors happen it's because Google changed something about their backend API and I have to wait for `ytmusicapi` to release a new version so I can update this `ytmusic-deleter` to use it. I see that there is a new release of `ytmusicapi` so I will cut a new release of `ytmusic-deleter` soon.

Having said that, I just ran `delete-uploads` on my system (Ubuntu 20.04) and I did not have that issue, although that doesn't necessarily mean anything. After I cut a new release I'll post a message here and I'll test in my Windows environment as well.
